docs/cudf/source/user_guide/pandas-comparison.md

```diff
@@ -139,6 +139,27 @@ module, which allow you to compare values up to a desired precision.
 Unlike Pandas, cuDF does not support duplicate column names.
 It is best to use unique strings for column names.
 
+## Writing dataframe to parquet with mixed column names
+
+When there is a dataframe with mixed column names, pandas type-casts each
+column name to `string` before writing to parquet file. `cudf` raises an
+error by default if this is attempted. However, to achieve similar behavior
+as pandas you can enable pandas compatibility mode option, which will
```

Comment:
```suggestion
as pandas you can enable the `mode.pandas_compatible` option, which will
```

Reply:
Please check that code snippets are using the right number of backticks. I think one backtick just italicizes the text…

Reply:
Verified that one backtick is bringing the code font from the deployed docs: https://docs.rapids.ai/api/cudf/nightly/user_guide/pandas-comparison/#no-true-object-data-type

Reply:
Of course. This is Markdown and not RST. My fault.